tangelo/problem_decomposition/dmet/dmet_problem_decomposition.py

```diff
@@ -105,6 +108,7 @@ def __init__(self, opt_dict):
                 setattr(self, k, v)
             else:
                 raise KeyError(f"Keyword :: {k}, not available in DMETProblemDecomposition.")
+        self.SecondQuantizedDMETFragment = SecondQuantizedDMETFragment
```

Comment:
We need another name for this attribute. `fragment_builder` or something like that.